authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,56 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
 
-"""Configure JWT Here"""
+from django.conf import settings
 
+from rest_framework import authentication, exceptions
+
+from .models import User
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    authentication_header_prefix = 'Token'
```

Comment:
@m-asiimwe There is an issue on this line. Please change authentication_header_prefix to 'Bearer and not 'Token'. This is because we will be passing in tokens in the format Bearer <token>.

Reply:
okay.. but is there any particular advantage Bearer gives us over Token?

Reply:
The team is okay with using 'Token' as long as it doesn't affect the functionality of the application which it doesn't